zebedee-cms/src/main/java/com/github/onsdigital/zebedee/model/publishing/Publisher.java

```diff
@@ -145,6 +143,17 @@ public static boolean executePublish(Collection collection, CollectionReader col
             success &= publishDatasets(collection);
         }
 
+        if (CMSFeatureFlags.cmsFeatureFlags().isStaticFilesPublishingEnabled()) {
+            try {
+                staticFilesServiceSupplier.getService().publishCollection(collection);
+                success &= true;
```

Comment:
Why are we doing `&=`, should it just not be `=`

Reply:
Its a logic =

So if success is null or true it will be set to true. if success is false it will stay false.

meaning that if something else has failed and set success to false it will stay as false until the end of the function.